src/platform/silabs/multi-ota/OTAMultiImageProcessorImpl.cpp

```diff
@@ -202,6 +202,11 @@ CHIP_ERROR OTAMultiImageProcessorImpl::SelectProcessor(ByteSpan & block)
 
 CHIP_ERROR OTAMultiImageProcessorImpl::RegisterProcessor(uint32_t tag, OTATlvProcessor * processor)
```

Comment:
We should input the enum instead of ta uint32

Reply:
Modified to take enum `OTAProcessorTag`